src/logics/morphoblue/configs.ts

```diff
@@ -5,8 +5,8 @@ type ContractNames = 'Morpho' | 'MorphoFlashLoanCallback';
 
 export interface MarketConfig {
   id: string;
-  loanTokenAddress: string;
```

Comment:
please also fix the `loanTokenAddress` in test/logics/morphoblue/helpers.ts

Reply:
fixed